pkgs/development/python-modules/insightface/default.nix

```
@@ -58,19 +58,24 @@ buildPythonPackage rec {
     tqdm
   ];
 
-  pythonImportsCheck = [
+  # aarch64-linux tries to get cpu information from /sys, which isn't available
+  # inside the nix build sandbox.
+  pythonImportsCheck = lib.optionals (stdenv.buildPlatform.system != "aarch64-linux") [
     "insightface"
     "insightface.app"
     "insightface.data"
   ];
-
-  passthru.tests.version = testers.testVersion {
-    package = insightface;
-    command = "insightface-cli --help";
-    # Doesn't support --version but we still want to make sure the cli is executable
-    # and returns the help output
-    version = "help";
-  };
+  passthru.tests.version =
+    if (stdenv.buildPlatform.system != "aarch64-linux") then
+      testers.testVersion {
```

```suggestion
  passthru.tests = lib.optionalAttrs stdenv.buildPlatform.system != "aarch64-linux" {
  version = testers.testVersion {
```

Otherwise, please use `versionCheckHook` and set `dontVersionCheck`.